packages/utils/recursive-parser.ts

```diff
@@ -15,7 +15,7 @@ export default function recursiveTransform(
 		console.error("Error in parsing top scope, Array required");
 		return false;
 	} else if (key === "merge") {
-		if (typeof value === "string") {
+		if (Array.isArray(value) && typeof value[0] === "string" && typeof value[1] === "string") {
```

Comment:
We can skip these comparrisons until later on in the import functions you've added. Checking for array is sufficient here, comparing the destructed values later